bookkeeper-dist/server/pom.xml

```diff
@@ -78,7 +78,7 @@
         <version>${maven-assembly-plugin.version}</version>
         <configuration>
           <finalName>bookkeeper-server-${project.version}</finalName>
-          <attach>false</attach>
+          <attach>true</attach>
```

Comment:
Then we wouldn't be testing what we actually ship.

How about we just skip the deploy phase for this module.

```
<plugin>
  <groupId>org.apache.maven.plugins</groupId>
  <artifactId>maven-deploy-plugin</artifactId>
  <version>X.Y</version>
  <configuration>
    <skip>true</skip>
  </configuration>
</plugin>
```

This will stop it being sent remotely, but sibling modules will be able to depend on it.

Reply:
That is fine to me as well.

Reply:
Just checked and this this configuration is already there